cabal.project

```diff
@@ -14,7 +14,7 @@ repository cardano-haskell-packages
 -- update either of these.
 index-state:
   -- Bump this if you need newer packages from Hackage
-  , hackage.haskell.org 2023-12-14T23:57:38Z
+  , hackage.haskell.org 2024-01-16T12:10:07Z
```

Comment:
`hackageNix` should be updated too

Reply:
Indeed but as Alex mentioned this has to wait 1 day

Reply:
Done